modules/engines/engine-hdb/src/main/java/com/sap/xsk/hdb/ds/processors/table/TableBuilder.java

```diff
@@ -51,7 +51,12 @@ public Table build(XSKDataStructureHDBTableModel model) {
   private HanaCreateTableBuilder createTableBuilder(String tableName, String tableType) {
     HanaSqlDialect dialect = new HanaSqlDialect();
     if (null != tableType) {
-      return SqlFactory.getNative(dialect).create().table(tableName, tableType);
+      if (tableType.equalsIgnoreCase(ISqlKeywords.KEYWORD_COLUMN))
+        return SqlFactory.getNative(dialect).create().table(tableName, ISqlKeywords.KEYWORD_COLUMNSTORE);
+      else if (tableType.equalsIgnoreCase(ISqlKeywords.KEYWORD_ROW))
+        return SqlFactory.getNative(dialect).create().table(tableName, ISqlKeywords.KEYWORD_ROWSTORE);
```

Comment:
This check is already done in https://github.com/eclipse/dirigible/blob/master/modules/database/database-sql/src/main/java/org/eclipse/dirigible/database/sql/dialects/hana/HanaCreateBranchingBuilder.java#L44